src/pages/ChatbotPage/index.tsx

```diff
@@ -2,15 +2,21 @@ import ChatbotIcon from "@/assets/ChatbotIcon.png";
 import { FaCircleChevronRight, FaChevronDown } from "react-icons/fa6";
 import { Message } from "./components/Messages";
 import Search, { SearchProps } from "antd/es/input/Search";
-import { useEffect, useRef, useState } from "react";
+import { useState } from "react";
 import { MessageProps } from "./components/Messages";
 import { postChatBotGenerateStream } from "./api";
 import { useToast } from "@/hooks/useToast";
 import useAuth from "@/hooks/useAuth";
+import { useChatScroll } from "./hooks";
+import { getUserAvatar } from "@/utils/userUtils";
+import { UserInfoType } from "@/types/userTypes";
+import avatar1 from "@/assets/avatar1@3x.png";
+import { CustomRedButton } from "@/components/Button";
+import { buttonData } from "./data";

```

Comment:
I would suggest to follow the import conventions: 

// Components 

// Assets 

// Constants - utils 

etc..